Formula/nettle.rb

```diff
@@ -4,8 +4,7 @@ class Nettle < Formula
   url "https://ftp.gnu.org/gnu/nettle/nettle-3.6.tar.gz"
   mirror "https://ftpmirror.gnu.org/nettle/nettle-3.6.tar.gz"
   sha256 "d24c0d0f2abffbc8f4f34dcf114b0f131ec3774895f3555922fe2f40f3d5e3f1"
-  # license ["GPL-2.0", "GPL-3.0", "LGPL-3.0"] - pending https://github.com/Homebrew/brew/pull/7953
-  license "GPL-2.0"
+  license any_of: ["GPL-2.0", "GPL-3.0", "LGPL-3.0"]
```

Comment:
```suggestion
  license any_of: ["GPL-2.0-or-later", "LGPL-3.0-or-later"]
```

From [the docs](http://www.lysator.liu.se/~nisse/nettle/nettle.html#Copyright):

> Nettle is dual licenced under the GNU General Public License version 2 or later, and the GNU Lesser General Public License version 3 or later.

I'm not sure we need to list `GPL-3.0` if it's included in the `GPL-2.0-or-later`. Another maintainer may want to weigh in here with their opinion.

Reply:
That seems to be the official position from the GNU project, indeed. From their [licenses FAQ page](https://www.gnu.org/licenses/gpl-faq.en.html#v2v3Compatibility):

> **Is GPLv3 compatible with GPLv2?**
>
> No. Many requirements have changed from GPLv2 to GPLv3, which means that the precise requirement of GPLv2 is not present in GPLv3, and vice versa. For instance, the Termination conditions of GPLv3 are considerably more permissive than those of GPLv2, and thus different from the Termination conditions of GPLv2.
>
> Due to these differences, the two licenses are not compatible: if you tried to combine code released under GPLv2 with code under GPLv3, you would violate section 6 of GPLv2.
>
> _However, if code is released under GPL “version 2 or later,” that is compatible with GPLv3 because GPLv3 is one of the options it permits._

I italicized the last sentence for emphasis, as it seems to address precisely your point.

Reply:
Also, from https://www.lysator.liu.se/~nisse/nettle/nettle.html#Copyright-1:

> Nettle is dual licenced under the GNU General Public License version 2 or later, and the GNU Lesser General Public License version 3 or later.

Your proposed change seems pretty solid in light of these remarks.